docs/src/format/table/index.md

```diff
@@ -183,18 +183,19 @@ See the `transaction.proto` file for its definition.
 The commit process is as follows:
 
 1. The writer finishes writing all data files.
-2. The writer creates a transaction file in the `_transactions` directory.
-   This file describes the operations that were performed, which is used for two purposes:
-   (1) to detect conflicts, and (2) to re-build the manifest during retries.
+2. Build a transaction struct based on the current version to tell how the manifest changes. Different operations have different structures for metadata changing.
 3. Look for any new commits since the writer started writing.
-   If there are any, read their transaction files and check for conflicts.
-   If there are any conflicts, abort the commit. Otherwise, continue.
-4. Build a manifest and attempt to commit it to the next version.
+   If there are any, begin the rebase process: read their transaction structures and check for conflicts.
+   If there are any conflicts and the conflicts are not retriable, abort the commit.
```

Comment:
We spell it with a `y` in the code, so easier for people to search for that if we use consistent spelling here.
```suggestion
   If there are any conflicts and the conflicts are not retryable, abort the commit.
```